msteams-platform/resources/samesite-cookie-update.md

```diff
@@ -80,6 +80,43 @@ Android WebView is a Chrome system component that allows Android apps to display
 > * You must declare first party cookies as `SameSite=Lax` or `SameSite=Strict`, as appropriate.
 > * You must declare third party cookies as `SameSite=None; Secure`.
 
+## Third party cookies deprecation
+
+All third party cookies set in the top-level domain are blocked when that domain is embedded in an iframe. This third party cookie deprecation is starting to roll out for all major browsers, with Chrome currently at a 1% rollout and other major browsers like Firefox and Safari to follow.
+
+This deprecation impacts a common scenario in which the external app is rendered within Teams in various entry points like personal apps, channel tabs, and conversational tabs.
+
+### Pop-out authentication scenario
+
+Pop-out authentication scenarios are a common method for apps to authenticate using different identity providers like Google Auth and Facebook Auth. Here's how it works:
```

Comment:
Looks like the official branded names of their auth services is [Facebook Login](https://developers.facebook.com/docs/facebook-login/) and [Google Identity](https://developers.google.com/identity/)

Reply:
Updated as suggested.